FreeRTOS/Test/CMock/smp/README.md

```diff
@@ -0,0 +1,162 @@
+# FreeRTOS Kernel SMP Unit Test
+
+FreeRTOS Kernel SMP unit test verifies the SMP scheduler logic in tasks.c which is enclosed by `configNUMBER_OF_CORES > 1`. The common scheduler logic for single core and SMP is still verified in FreeRTOS/FreeRTOS/Test/CMock/tasks.
```

Comment:
Can we break the lines at 80 chars like in the `### Coverage tests` section?

Reply:
Update in new commit.